### ISSUE TITLE
Font size to choose

### ISSUE THREAD

**Issue by hhblaze** [alexwforsythe/code-blocks#16]
Hi, is it possible to choose the font-size?

**Comment by alexwforsythe** [comment]
You can choose your font size as you would normally in docs.

The highlighting function shouldn't change the original size of the selection, and you can adjust the font size normally after highlighting it.

**Comment by hhblaze** [comment]
Yes, that's clear. In my doc normal text and code have different font-sizes, it means that each pasted code must be separately set to the new font-size value...and it's annoying... 
If it could b possible to choose font size of the code-block from combobox, like "Theme", "Language" even "No background" checkbox...it would make the life brighter!
I think it's a nice feature.

Thank you!

**Comment by iraklisg** [comment]
:+1: 
Indeed, defining font-size for code-blocks would be very helpfull

**Comment by ginigangadharan** [comment]
Any update for this issue ?
